src/lib.rs

```diff
@@ -267,6 +267,39 @@ fn run_tests_threaded<D: 'static + Send + Sync>(
     recv
 }
 
+/// Run the given tests one by one in the current thread.
+fn run_tests_serialized<D: 'static + Send + Sync>(
+    args: &Arguments,
+    tests: Vec<Test<D>>,
+    run_test: impl Fn(&Test<D>) -> Outcome + 'static + Send + Sync,
+) -> impl IntoIterator<Item = RunnerEvent<D>> {
+    let args_ignored = args.ignored;
+    let args_test = args.test;
+    let args_bench = args.bench;
+    let run_test = std::sync::Arc::new(run_test);
+    tests.into_iter().flat_map(move |test| {
+        let run_test = run_test.clone();
+        std::iter::once(RunnerEvent::Started {
+            name: test.name.clone(),
+            kind: test.kind.clone()
+        })
+        .chain(std::iter::once_with(move || {
```

Comment:
I really can't stand how rustfmt formats this :/
The fact that `.chain` has the same indentation as everything else in the block despite continuing an expression throws me off every time.

 Let's work around that like this:

```suggestion
        let start_event = RunnerEvent::Started {
            name: test.name.clone(),
            kind: test.kind.clone()
        };
        std::iter::once(start_event).chain(std::iter::once_with(move || {
```